src/core/scrobbler/listenbrainz/listenbrainz-scrobbler.ts

```diff
@@ -21,8 +21,6 @@ const apiUrl = `${baseUrl}/submit-listens`;
 export default class ListenBrainzScrobbler extends BaseScrobbler<'ListenBrainz'> {
 	public userApiUrl!: string;
 	public userToken!: string;
-	/** Can't find where this value is being set from */
-	private authUrl!: string;
```

Comment:
I think it was changed in https://github.com/web-scrobbler/web-scrobbler/commit/9f885e4c4d31d03e64b497b68f9b3a3e901ea1e2#diff-e2b6cac4e0d320435974b23d0ebc5174fb22df607c8def762f90dd7eb1004830L272 and never cleaned up